test/e2e/tests/dapp-interactions/revoke-permissions.spec.js

```diff
@@ -25,12 +24,10 @@ describe('Wallet Revoke Permissions', function () {
         // Get initial accounts permissions
         await driver.clickElement('#getPermissions');
 
-        const permissionsResult = await driver.findElement(
-          '#permissionsResult',
-        );
-
-        // Eth_accounts permission
-        assert.equal(await permissionsResult.getText(), 'eth_accounts');
```

Comment:
wrong pattern, which leads to race conditions. We should find the we want element by the expected text